operators/advanced_network/python/tune_system.py

```diff
@@ -347,14 +407,26 @@ def check_nvidia_gpu_clocks():
 
             logging.debug(f"GPU {idx}: Checking clocks...")
 
-            if sm_current != sm_max:
+            # Some GPUs have a boost clock that appears as the "max clock", but when you set the
+            # GPU to that frequency it will not report as that with nvidia-smi. For example:
```

Comment:
Did you confirm this? Do we have a way to confirm it otherwise?

Reply:
I confirmed that at least on data center cards this is true, but I'm trying to confirm with RTX. I believe it's true.

Reply:
as in someone told you, you found some specs, or you have a programmatic way to check?